src/extension.ts

```diff
@@ -580,7 +578,6 @@ function launchMetals(
       scalaDebugger
         .initialize(outputChannel)
         .forEach(disposable => context.subscriptions.push(disposable));
-      registerCommand(scalaDebugger.startSessionCommand, scalaDebugger.start);
```

Comment:
@marek1840 the command is now automatically registered by `ClientCommands` (see the relative fix above). Can you double check this is ok?

Reply:
I've tested that I can still use the extension to run a main, as expected.

Reply:
Thanks for testing!